lib/syskit/network_generation/dataflow_dynamics.rb

```diff
@@ -620,10 +620,9 @@ def policy_for(
                 end
 
                 source_port_m = source_port.model
-                unless source_port_m.init_policy.nil?
-                    policy.merge(init: source_port_m.init_policy)
-                end
-                policy
+                return policy unless source_port_m.init_policy?
```

Comment:
Once more. Always set the policy. It makes no sense to not set it to false in this case.